force-app/main/default/classes/CON_ContactMerge_CTRL.cls

```diff
@@ -627,8 +627,16 @@ public with sharing class CON_ContactMerge_CTRL {
 
         // Sort the custom fields list by name before creating the list of FieldRow for merge UI
         List<String> customFieldList = new List<String>();
-        customFieldList.addAll(customFieldMap.keySet());
+        String undeliverableFieldName = String.valueOf(Contact.Undeliverable_Address__c).toLowerCase();
```

Comment:
This block of code could be extracted to a function called `buildSortedCustomFieldList`.

Reply:
good callout